widget/markdown.go

```diff
@@ -29,6 +29,11 @@ func (t *RichText) ParseMarkdown(content string) {
 
 // AppendMarkdown parses the given markdown string and appends the
 // content to the widget, with the appropriate formatting.
+// This API is intended for appending complete markdown documents or
+// standalone fragments, and should not be used to parse a single
+// markdown document piecewise. Use ParseMarkdown for this case.
```

Comment:
The indication that "ParseMarkdown" can work piecewise doesn't seem correct - that code is being used within this helper so I don't know how it could be true.